packages/app/pages/index.tsx

```diff
@@ -1,29 +1,48 @@
-import { User } from '@koh/common'
-import Router from 'next/router'
 import { ReactElement } from 'react'
-import { useHomePageRedirect } from '../hooks/useHomePageRedirect'
-import { useProfile } from '../hooks/useProfile'
-import { Spin } from 'antd'
+import { Button, Image, List } from 'antd'
+import styled from 'styled-components'
+
+const Container = styled.div`
```

Comment:
Considering using this component instead

``` typescript
import { StandardPageContainer } from '../components/common/PageContainer'
```

Reply:
this component did not really work, plus it seems like the current page looks fine on mobile resolutions. Though, I will convert the container to just a div with tailwind since it's just supposed to be a one time container anyway